pages/index.tsx

```diff
@@ -1,21 +1,37 @@
 import { Box } from '@chakra-ui/react'
 import { useEffect } from 'react'
 import type { NextPage } from 'next'
-import Head from 'next/head'
 import Hero from '../components/Hero'
 import * as FullStory from '@fullstory/browser'
+import { NextSeo } from 'next-seo'
 
 const Home: NextPage = () => {
   useEffect(() => {
     FullStory.init({ orgId: 'o-1CKVPB-na1' })
   }, [])
   return (
     <div>
-      <Head>
-        <title>Developer DAO School of Code</title>
-        <meta name="description" content="Developer DAO's school of code" />
-        <link rel="icon" href="/favicon.ico" />
-      </Head>
+      <NextSeo
+        title="Developer DAO School of Code"
+        description="Developer DAO's school of code"
+        openGraph={{
+          images: [
+            {
+              url: '/landing-page-screenshot.png',
```

Comment:
I think technically this needs to be an absolute url. We could just use <https://school-of-code.vercel.app/landing-page-screenshot.png> for now.

Here's a bit more info I found: <https://veonr.com/blog/relative-vs-absolute-og-image-video-urls>

Reply:
Nice catch. I thought Vercel would replace this at build time with the VERCEL_URL (It should IMO)

Also had to add this env var to get this to work.

It should be automatically exposed. More on that here: https://vercel.com/docs/concepts/projects/environment-variables#system-environment-variables

If not, whoever has access to the Vercel dashboard should check it.

Reply:
Hey @traversng this is very close, I noticed just one last item. VERCEL_URL doesn't actually add the `https://` which is a pain since we can't just hard-code `https://` since that'll break localhost.

Though since we can't preview the image on localhost anyhow (using online validators like opengraph.xyz), how about you just hard-code `https://` in React for now?

And in that case, maybe we should just pull out the whole `.env.local` setup instructions and example file, since it's ok if this doesn't work outside of live deploys. In other words, we can just stick with the default of "" for VERCEL_URL running on localhost.

Though I really appreciated that you made the extra effort to document the env setup!

But your choice, if you'd rather leave it, that's ok too.

Reply:
ps: Vercel builds a preview site for each PR, so you can check that the final path looks good using the preview too!

Reply:
Hey @traversng here's just a quick ping. This is very close to mergin'!

It really just needs `https://` prefix!

And then what do you think about not worrying about setting up the local `.env.local` (since it's ok to not work locally since it just impacts metadata)?

Reply:
(though if we were to leave in `.env.local` we'd need to add another env var for protocol, since `http/https` will have to be changed depending on env)

Reply:
Hey @traversng I add `https://` and removed .env for now to keep things simple. You mentioned maybe another way to solve. I'll merge this for MVP, but please open a new PR if there are changes you want to make down the road.